n2v/inverter.py

```diff
@@ -200,18 +204,18 @@ def diagonalize(self, matrix, ndocc):
 
     #------------->  Inversion:
 
-    def invert(self, method="wuyang", 
+    def invert(self, method,
                      opt_method='trust-krylov', 
-                     potential_components = ["fermi_amaldi", "svwn"], 
+                     guide_potential_components = ["fermi_amaldi", "svwn"], 
```

Comment:
This was my original name, and I spent an unnecessary amount of time deciding what to call this since I thought it was super long. I don't know if something like "vpot_guide" or "v_components" or anything else more compact would work. Otherwise, this is fine.